### PR TITLE
[Review Only] Update function common starter

### PR DESCRIPTION
**Pull request by ilayaperumalg**

### CODE REVIEW

File: http-app-dependencies/pom.xml [hunk 1]
```diff
@@ -4,24 +4,43 @@
...
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>app-starters-function-common</artifactId>
+				<version>2.0.3.BUILD-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-function-core</artifactId>
+				<version>${spring-cloud-function.version}</version>
```
Comment (artembilan) [inline http-app-dependencies/pom.xml:38]: That's something wrong. I though SCSt 2.1 already depends on SC-function. Therefore we don't need these dependencies here.

Any explanations, please?

File: http-app-dependencies/pom.xml [hunk 2]
```diff
@@ -4,24 +4,43 @@
...
 			<dependency>
 				<groupId>org.springframework.cloud.stream.app</groupId>
 				<artifactId>spring-cloud-starter-stream-source-http</artifactId>
-				<version>2.0.2.BUILD-SNAPSHOT</version>
+				<version>2.0.3.BUILD-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>app-starters-function-common</artifactId>
+				<version>2.0.3.BUILD-SNAPSHOT</version>
```
Comment (artembilan) [inline http-app-dependencies/pom.xml:33]: I don't think we need this dependency here, especially with the version.
See similar approach for all the `org.springframework.cloud.stream.app` deps in the SFTP, for example: https://github.com/spring-cloud-stream-app-starters/sftp/blob/master/spring-cloud-starter-stream-source-sftp/pom.xml#L29

You see? That comes without version and only in the target module, but definitely not root...

**Review by artembilan — CHANGES_REQUESTED** [review]

File: http-app-dependencies/pom.xml (hunk 1 above)
Comment (olegz) [reply http-app-dependencies/pom.xml:38]: That is correct, SCF should be brought in by SCSt

**Comment by sabbyanandan** [comment]
@ilayaperumalg: This is not necessary anymore - right? Could you confirm, please?

**Comment by sabbyanandan** [comment]
Closing this due to no activity.

**Pull request closed without merging**